layers/+intl/japanese/README.org

```diff
@@ -0,0 +1,74 @@
+#+TITLE: Japanese layer
+#+HTML_HEAD_EXTRA: <link rel="stylesheet" type="text/css" href="../../css/readtheorg.css" />
+
+
+* Table of Contents                                         :TOC_4_org:noexport:
+ - [[Description][Description]]
+ - [[Install][Install]]
+   - [[Layer][Layer]]
+   - [[Configuration][Configuration]]
+     - [[migemo][migemo]]
+     - [[avy-migemo][avy-migemo]]
+     - [[ddskk][ddskk]]
+     - [[pangu-spacing][pangu-spacing]]
+ - [[Key Bindings][Key Bindings]]
+
+* Description
+This Layer adds Japanese related packages:
+- [[https://github.com/emacs-jp/migemo][migemo]]: Japanese incremental search through dynamic pattern expansion
+- helm with migemo
+- [[https://github.com/momomo5717/avy-migemo/blob/master/README.jp.org][avy-migemo]]: avy with migemo
+- [[https://github.com/hsaito/ddskk][ddskk]]: Simple Kana to Kanji conversion program (SKK)
+- [[https://github.com/emacs-jp/japanese-holidays][japanese-holidays]]: calendar functions for the Japanese calendar
+- [[https://github.com/coldnew/pangu-spacing][pangu-spacing]]: emacs minor-mode to add space between Japanese and English characters.
+- Join consecutive Japanese lines into a single long line without unwanted space when exporting org-mode to html.
+  
+* Install
+** Layer
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =japanese= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+** Configuration
+For a basic configuration, add this to your =~/.spacemacs= inside ~dotspacemacs/user-config~:
```

Comment:
Please read [conventions on spacing in documentation](https://github.com/syl20bnr/spacemacs/blob/develop/doc/CONVENTIONS.org#spacing-in-documentation). Not critical, but anyway.